src/cli/commands/crossmsg/mod.rs

```diff
@@ -31,7 +31,7 @@ impl CrossMsgsCommandsArgs {
             Commands::Fund(args) => Fund::handle(global, args).await,
             Commands::Release(args) => Release::handle(global, args).await,
             Commands::Propagate(args) => Propagate::handle(global, args).await,
-            Commands::WhitelistPropagator(args) => WhitelistPropagator::handle(global, args).await,
+            Commands::SendCrossMsgs(args) => SendCrossMsg::handle(global, args).await,
```

Comment:
Hmm.. `SendCross` was supposed to be an internal function from the smart contract not accessible by externally owned accounts, but now that I see it here I think it is a good idea to also expose it in the agent. It may come handy for some users.

Reply:
maybe just RPC calls then